process/block/postprocess/feeHandler.go

```diff
@@ -36,17 +39,34 @@ func (f *feeHandler) GetAccumulatedFees() *big.Int {
 }
 
 // ProcessTransactionFee adds the tx cost to the accumulated amount
-func (f *feeHandler) ProcessTransactionFee(cost *big.Int) {
+func (f *feeHandler) ProcessTransactionFee(cost *big.Int, txHash []byte) {
```

Comment:
`txHash` will be nil for some calls. Do we need to handle this differently?

Reply:
txHash will not be nil, should not be nil. I added everywhere.